apps/swirl-docs/src/components/Layout/CategoryNav.tsx

```diff
@@ -25,18 +31,84 @@ export function CategoryNav() {
   //   </ul>
   // );
 
-  const SubElement = ({ navItem }: { navItem: NavItem }) => (
-    <li
-      className={classNames("font-sm mb-4", "hover:text-border-info", {
-        "text-border-info": activePath === navItem.url,
-        "text-text-default": activePath !== navItem.url,
-      })}
-    >
-      <Link href={`${navItem.url}`}>
-        <a className="text-sm capitalize">{navItem.title}</a>
-      </Link>
-    </li>
-  );
+  const SubElement = ({ navItem }: { navItem: NavItem }) => {
+    const [isExpanded, setIsExpanded] = useState(false);
+
+    useEffect(() => {
+      if (activePath.includes(navItem.url)) {
+        setIsExpanded(true);
+      }
+    }, [navItem.url]);
+
+    return (
+      <li className={classNames("mb-2 py-2", { "max-h-10": !isExpanded })}>
+        <div className="flex justify-between items-center">
+          <Link href={`${navItem.url}`}>
+            <a
```

Comment:
**nitpick:** Could you make the link fill the available width? There is an awkward gap between the link and the toggle which is non interactive.

Reply:
![Bildschirm­foto 2023-05-03 um 16 15 26](https://user-images.githubusercontent.com/23700195/235942702-12ce5c25-ba76-4bff-9516-b759c98ca7d9.png)

do you mean this one?

Reply:
ah that link! alright :)